src/puppetlabs/trapperkeeper/services/status/status_proxy_service.clj

```diff
@@ -0,0 +1,16 @@
+(ns puppetlabs.trapperkeeper.services.status.status-proxy-service
+  (:require [clojure.tools.logging :as log]
+    [puppetlabs.trapperkeeper.core :refer [defservice]]))
+
+(defservice status-proxy-service
+  [[:WebroutingService add-proxy-route]
+   [:ConfigService get-in-config]]
+  (init [this context]
+    (add-proxy-route
+      this
+      {:host (get-in-config [:status-proxy :target-host])
```

Comment:
This sounds good to me. And the `:scheme :https` bit exposes the implementation, which would make it difficult to change the config later too.